src/client.rs

```diff
@@ -200,6 +250,19 @@ impl Client {
         self.issue(WebDriverCommand::Refresh).await?;
         Ok(())
     }
+
+    /// Get the current page title.
+    ///
+    /// See [9.6 Get Title](https://www.w3.org/TR/webdriver1/#dfn-get-title) of the WebDriver standard.
+    #[cfg_attr(docsrs, doc(alias = "Get Title"))]
+    pub async fn title(&mut self) -> Result<String, error::CmdError> {
+        let title = self.issue(WebDriverCommand::GetTitle).await?;
+        if let Some(title) = title.as_str() {
```

Comment:
Let's use `let Some(Value::String(s)) = title { return s }` here instead to avoid unnecessarily allocating a new string :+1: